src/main/java/appeng/items/tools/powered/ToolWirelessTerminal.java

```diff
@@ -56,6 +57,50 @@ public ToolWirelessTerminal()
 	{
 		super( AEConfig.instance.wirelessTerminalBattery, Optional.<String>absent() );
 		this.setFeature( EnumSet.of( AEFeature.WirelessAccessTerminal, AEFeature.PoweredTools ) );
+
+		if( Platform.isClient() )
+		{
+			MinecraftForgeClient.registerItemRenderer( this, new ToolWirelessTerminalRender() );
+		}
+	}
+
+	public static AEColor getColor( ItemStack item )
+	{
+		final NBTTagCompound c = item.getTagCompound();
+		if( c != null && c.hasKey( "color" ) )
+		{
+			String color = c.getString( "color" );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'color' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)